client/components/AppMap.vue

```diff
@@ -50,30 +56,65 @@ export default {
       showCard: false,
       cardContent: '',
       cardPosition: { x: '50px', y: '20px' },
+      //mapObject: {},
+      //mapKey:0,
+      //mapBounds:L.latLngBounds(L.latLng(40.712, -74.227), L.latLng(40.774, -74.125)),
     };
   },
   methods: {
     showInfoCard(event, content) {
-    this.cardContent = content;
-    this.$nextTick(() => {
+      this.cardContent = content;
+      this.$nextTick(() => {
         this.showCard = true;
-    });
-}
-
-,
+      });
+    },
     hideInfoCard() {
       this.showCard = false;
     },
     navigateToDetails(id) {
       this.$router.push(`/details-page?id=${id}`);
     },
-  }
-  ,
+    computeBoundsZoom() {
+      
+      this.$nextTick(() => {
+        this.$nextTick(()=>{
+          setTimeout((a) => {
+            // console.log(a);
+            // console.log(this.$refs.map);
+            // console.log(this.$refs.map.leafletObject);
+            let mapObject = this.$refs.map.leafletObject;
+            mapObject.fitBounds(this.bounds);
+            this.zoom = mapObject.getBoundsZoom(this.bounds);
+            mapObject.setView([this.latitude, this.longitude], this.zoom);
+          }, 3000);
+        })
+      })
+    },
+    // onReady(mapObject) {
+    //   this.zoom = mapObject.getBoundsZoom(this.bounds);
+    //   console.log("onReady");
+    //   console.log(this.zoom);
+    //   // use the mapObject to call native Leaflet methods
+    //   mapObject.on("moveend", function(e) {
+    //     console.log(e);
+    //   });
+    // },
+    // onUpdateBounds({ bounds }) {
+    //   // assign the center coordinates to the data properties
+    //   this.bounds = bounds;
+    //   console.log(bounds);
+    // }
```

Comment:
will these be used in the future? maybe we can remove these.

Reply:
I removed these lines for clarity.